### ISSUE TITLE
doesnt work with zf 2.2.1

### ISSUE THREAD

**Issue by Perfect-Web** [ZF-Commons/ZfcTwig#69]
I get this

Zend\ServiceManager\Exception\ServiceNotCreatedException: An exception was raised while creating "ZfcTwigViewStrategy"; no instance returned in ...
 Zend\ServiceManager\Exception\ServiceNotCreatedException: An exception was raised while creating "ZfcTwigRenderer"; no instance returned in....
Zend\ServiceManager\Exception\ServiceNotCreatedException: An exception was raised while creating "ZfcTwigEnvironment"; no instance returned in ....
Zend\ServiceManager\Exception\ServiceNotCreatedException: An exception was raised while creating "ZfcTwigLoaderChain"; no instance returned in .....

and that is all on one page.

Basically it says that the instance is not returned however i have checked the module.php and all should be correct.

I have to mention that the install was done thru composer
